repotracker/repotracker.go

```diff
@@ -931,6 +931,7 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 			// add only tasks that require activation times
 			for _, bvt := range buildvariant.Tasks {
 				tId, ok := taskNameToId[bvt.Name]
+				// TODO: remove activation check
```

Comment:
Keeping this activation check in for now so that we only log the debug statement for the cases I didn't already know about; will remove this when I remove the logging